src/rules/__tests__/prefer-comparison-matcher.test.ts

```diff
@@ -145,7 +145,6 @@ const testComparisonOperator = (
       `expect(value).${preferredMatcherWhenNegated}(1);`,
       `expect(value).not.${preferredMatcher}(1);`,
       `expect(value).not.${preferredMatcherWhenNegated}(1);`,
-      `expect(value).${preferredMatcher}(1);`,
```

Comment:
?

Reply:
Turns out it's a duplicate: https://github.com/jest-community/eslint-plugin-jest/pull/1113/files#diff-0be7671b0597487883f125f6da4e56cf1661f717eee3af6412ae834a95a416e0L144

Reply:
ah, cool 👍